Content.Server/Cargo/CargoOrderDatabase.cs

```diff
@@ -86,9 +90,16 @@ public bool RemoveOrder(int orderNumber)
         /// <param name="order">The order to be approved.</param>
         public void ApproveOrder(int orderNumber)
         {
+            if (CurrentOrderSize == MaxOrderSize)
+                return;
             if (!_orders.TryGetValue(orderNumber, out var order))
                 return;
+            else if (CurrentOrderSize + order.Amount > MaxOrderSize) { 
```

Comment:
1. Please move opening curly bracket to the next line.
2. As for me it will be more clear if you use additional pair of the brackets for the `if` condition:

```csharp
else if ((CurrentOrderSize + order.Amount) > MaxOrderSize)
{
    ...
}
```